src/avro2json.c

```diff
@@ -401,6 +407,21 @@ static int avro_value_to_json_t(const avro_value_t *value, json_t **json,
     const void *val;
     size_t size;
     CHECKED_EV(avro_value_get_fixed(value, &val, &size));
+
+    if(conf->ms_hadoop_logical_types && size == 16){
```

Comment:
```suggestion
    if (conf->ms_hadoop_logical_types && (size == 16)) {
```

Reply:
nit: please go over all the places you added, and add spaces where needed, like I did above.